DatadogSessionReplay/Tests/Utilities/SwiftExtensionsTests.swift

```diff
@@ -44,6 +44,13 @@ class FixedWidthIntegerTests: XCTestCase {
         let expectedConvertedValue = Int.min
         XCTAssertEqual(expectedConvertedValue, convertedValue)
     }
+
+    func testBytesSizeConvenience() {
+        let size = 1
+        XCTAssertEqual(size.KB, size * 1_024)
+        XCTAssertEqual(size.MB, size * 1_024 * 1_024)
+        XCTAssertEqual(size.GB, size * 1_024 * 1_024 * 1_024)
+    }
```

Comment:
We add these extensions as part of `Sources/Datadog/Utils/SwiftExtensions.swift`, but this test file relates to Session Replay's extensions. It needs to be moved 😉.